freqtrade/commands/cli_options.py

```diff
@@ -425,6 +425,11 @@ def __init__(self, *args, **kwargs):
         choices=["DB", "file"],
         default="file",
     ),
+    "trade_ids": Arg(
+        '--tradeids',
```

Comment:
```suggestion
        '--trade-ids',
```

-- to be aligned with other options, like `db-url`, which have `-` in the name...
(fix all other appearances of this option as well)